src/Templates/ProjectTemplates/CSharp/.NETCore/ConsoleApplication/NuGet.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0"
```

Comment:
Should include "type: platform" to match the current "dotnet new" template in .net core.